src/localization/es/learn.json

```diff
@@ -0,0 +1,163 @@
+
+  "steps.rangeNumber.title": "Rango de números `[0-9]`",
+  "steps.rangeNumber.description": "Para encontrar números en un rango especificado, el número de inicio y el de final se escriben dentro de corchetes `[]` con un guíon `-` entre ellos. Escriba una expresión que seleccionará todos los números entre `3` y `6`, incluidos ellos.",
+
+  "examples.basicMatchers.title": "Practica: Matcher básico",
+  "examples.basicMatchers.description": "Escriba la expresión que seleccionará las letras `of` en el texto.",
+
+  "examples.dotCharacter.title": "Practica: Cualquier carácter",
+  "examples.dotCharacter.description": "Escriba la expresión para seleccionar letras, números, espacios y caracteres especiales individuales en el texto. La expresión que escribe debe coincidir con cualquier carácter.",
+
+  "examples.characterSet.title": "Practica: Conjunto de caracteres",
+  "examples.characterSet.description": "Escriba la frase que coincida con cada palabra en el texto. Los únicos caracteres que cambian son las iniciales de las palabras.",
+
+  "examples.negatedCharacterSet.title": "Practica: Conjunto de caracteres negados",
+  "examples.negatedCharacterSet.description": "Escriba la expresión que coincidirá con cualquier otra cosa que no sea las palabras `beor` y `beur` en el texto. Haz esto usando el conjunto de caracteres negados.",
+
+  "examples.range.title": "Practica: Rango de letras",
+  "examples.range.description": "Escribe la expresión que seleccionará las letras de `g` a `k` en el texto.\\nlas letras `g` y `k` también deben incluirse en el rango.",
+
+  "examples.rangeNumber.title": "Practica: Rango de letros",
+  "examples.rangeNumber.description": "Escriba la expresión que seleccionará los números de `2` a `7` en el texto.\\n`2` and `7` también deben incluirse en el rango.",
+
+  "steps.repetitions.title": "Repeticiones",
+  "steps.repetitions.description": "Algunos caracteres especiales se utilizan para especificar cuántas veces se repetirá un carácter en el texto. Estos caracteres especiales son el más `+`, el asterisco `*`, y el signo de interrogación `?`.",
+
+  "steps.asterisk.title": "Asterisco `*`",
+  "steps.asterisk.description": "Ponemos un asterisco `*` al inicio para indicar que un carácter no coincide en absoluto o puede coincidir muchas veces. Por ejemplo, indica que la letra `e` nunca debe ocurrir en el texto, o puede ocurrir una o más de lado a lado.",
+
+  "steps.plusCharacter.title": "Signo más `+`",
+  "steps.plusCharacter.description": "Para indicar que un carácter puede ocurrir una o más veces, ponemos el signo más `+` al final. Por ejemplo, indica que la letra `e` puede ocurrir una o más veces en el texto.",
+
+  "steps.questionMark.title": "Signo de interrogación `?`",
+  "steps.questionMark.description": "Para indicar que un carácter es opcional, ponemos un signo de interrogación `?` al final. Por ejemplo, indica que la siguiente letra `u` es opcional.",
+
+  "steps.quantifier.title": "Llaves - 1",
```

Comment:
Isn't `"Llaves"` actually `"Tirantes Rizados"`?    
`{}`

Reply:
"_Tirantes Rizados_" is not a thing in spanish, here you can find the names in spanish:
- parenthesis = paréntesis
- curly braces = llaves
- brackets = corchetes

![image](https://user-images.githubusercontent.com/315504/145095538-fcf489e1-7ff1-4454-822e-0e1c534e33c4.png)